src/modules/main/main-shell/Breadcrumbs.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { useLocation, useNavigate } from "react-router-dom";
+import { Breadcrumbs as MUIBreadcrumbs, Link, Typography } from '@mui/material';
```

Comment:
Codacy has a fix for the issue: [Replace `'@mui/material'` with `"@mui/material"`](https://app.codacy.com/gh/abdulqhayum47/lets-react/pullRequest?prid=9776931)
```suggestion
import { Breadcrumbs as MUIBreadcrumbs, Link, Typography } from "@mui/material";
```